x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/cases.test.ts

```diff
@@ -110,76 +112,30 @@ describe(`cases`, () => {
 
         expect(casesFeaturePrivilege.getActions(privilege, feature)).toMatchInlineSnapshot(`
           Array [
-            "cases:1.0.0-zeta1:security/getCase",
-            "cases:1.0.0-zeta1:security/getComment",
-            "cases:1.0.0-zeta1:security/getTags",
-            "cases:1.0.0-zeta1:security/getReporters",
-            "cases:1.0.0-zeta1:security/getUserActions",
-            "cases:1.0.0-zeta1:security/findConfigurations",
-            "cases:1.0.0-zeta1:security/createCase",
-            "cases:1.0.0-zeta1:security/deleteCase",
-            "cases:1.0.0-zeta1:security/updateCase",
             "cases:1.0.0-zeta1:security/pushCase",
+            "cases:1.0.0-zeta1:security/createCase",
             "cases:1.0.0-zeta1:security/createComment",
-            "cases:1.0.0-zeta1:security/deleteComment",
-            "cases:1.0.0-zeta1:security/updateComment",
             "cases:1.0.0-zeta1:security/createConfiguration",
-            "cases:1.0.0-zeta1:security/updateConfiguration",
-          ]
-        `);
-      });
-
-      it('grants both `all` and `read` privileges under feature', () => {
```

Comment:
Is this case covered from the first test in the file?

Reply:
This one is covered by the test called `'grants all privileges under feature'`. The old test that had that same name is covered by the first test that does the `it.each`.